src/tests/ftest/dfuse/daos_build.py

```diff
@@ -127,7 +127,7 @@ def run_build_test(self, cache_mode, intercept=False, dfuse_namespace=None):
         cache_time = '2d'
         # Timeout in minutes.  This is per command so up to double this or more as there are two
         # scons commands which can both take a long time.
-        build_time = 10
+        build_time = 60
```

Comment:
This seems like total overkill, this test often runs in < 10 minutes and has to run two sizeable scons commands so we can assume that each of them is taking < 5 minutes in many cases.  A 60 minute timeout seems far too long.

We're already recently increased the time by about 12x for some versions.

Looking at the code this will only affect writeback, metadata and writethrough when IL not enabled, is that correct?

Reply:
The `build_time` is only used to set the base value for the scons commands timeouts - the `build_time` is multiplied by 60 to set the timeout for scons commands only - all other commands use a 600 second timeout (per L213-215).

The following tests were previously using and are now using the following base scons command timeouts:
```
- 'writeback':                    previous=60,  now=60
- 'writethrough' w/o intercept:   previous=10,  now=60  <== increase
- 'writethrough' w intercept:     previous=360, now=360
- 'metadata':                     previous=10,  now=60  <== increase
- 'data':                         previous=120, now=120
- 'nocache':                      previous=300, now=300
```

This was the request made in https://daosio.atlassian.net/browse/DAOS-12724?focusedCommentId=111734

Reply:
The writethrough w/o intercept took 16 minutes for the deps build and 1:30 for the daos build.  The metadata test took 19 minutes for the deps build and and the daos build took four minutes.

So yes, this test would have failed with the previous settings and now has a 3x margin.